.github/workflows/uberjar.yml

```diff
@@ -0,0 +1,77 @@
+name: Uberjar
+
+on:
+  push:
+    branches:
+      - master
+      - 'release-**'
+    paths-ignore:
+    - 'docs/**'
+
+jobs:
+
+  build:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        edition: [ee, oss]
+    env:
+      MB_EDITION: ${{ matrix.edition }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Prepare Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+    - name: Prepare JDK 8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: Install gettext
+      run: sudo apt install gettext
+    - name: Install Clojure CLI
+      run: |
+        curl -O https://download.clojure.org/install/linux-install-1.10.1.708.sh &&
+        sudo bash ./linux-install-1.10.1.708.sh
+    - name: Check versions
+      run: |
+        echo "Node.js `node --version`"
+        echo "yarn `yarn --version`"
+        java -version
+        echo "Clojure `clojure -e "(println (clojure-version))"`"
+        lein --version
+        msgfmt --version
+
+    - name: Get yarn cache
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/yarn
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+    - run: yarn install --lockfile
+
+    - name: Get M2 cache
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/project.clj') }}-${{ hashFiles('**/deps.edn') }}
+    - run: lein with-profile +include-all-drivers,+cloverage,+junit,+oss deps
+
+    - run: ./bin/build version
+    - run: ./bin/build translations
+    - run: ./bin/build frontend
+    - run: ./bin/build drivers
+    - run: ./bin/build uberjar
```

Comment:
these can all be ran with one command e.g.

```
./bin/build version translations frontend drivers uberjar
```

Those are the default steps tho, so you can simply run `./bin/build` instead.

Reply:
Oh, that's good explanation. I was wondering what's the difference between `.bin/build` and the 4 of these scripts running sequentially.

Reply:
The separation was done so we can look at the execution time of the individual step. But I guess collapsing them is also fine.

![image](https://user-images.githubusercontent.com/7288/101547263-d8169600-395e-11eb-86de-e5b48e061e06.png)

Reply:
Well, that's also a good call, @ariya 🤔